content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -94,54 +96,62 @@ Proceed to one of the options for connecting the remote cluster to the local clu
     --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} > $HOME/istio-remote.yaml
     {{< /text >}}
 
-1.  Create a namespace for remote Istio.
+1.  Create an `istio-system` namespace for remote Istio with the following
```

Comment:
It might be useful to note that the namespace for the istio components should be the same on all clusters.  The reason I say this is it is possible to override the "istio-system" name on the main cluster although that is rarely done.

Reply:
I am reluctant to expose that capability if it is rarely used. The content tries hard to stick to a single narrative to make things easier for the readers. That said, I do agree that a note stating that the namespace should be the same for all Istio components might be helpful for readers with existing clusters.

Reply:
Stating they should be the same is sufficient IMO.   FWIW I don't know how common it is for people to change the namespace in production deployments anyway.

Reply:
Added a note with your comments.